apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -29,10 +32,41 @@ const useSections = () => {
   return [sections, updateSection];
 };
 
+const saveSection = (e, section) => {
+  e.preventDefault();
+
+  const form = document.querySelector(`#${FORM_ID}`);
+  if (!form.checkValidity()) {
+    form.reportValidity();
+    return;
+  }
+
+  const csrfToken = document.querySelector('meta[name="csrf-token"]')
+    .attributes['content'].value;
+
+  fetch(SECTIONS_API, {
```

Comment:
i went with the ajax submit here because we have all the data in the components and it seemed both easy to do now, and easy to extend later if/when we do multiple sections at once. but as far as I can tell, there's still nothing preventing us from doing a regular form submit instead, if we wanted to.